pkg/reconciler/build/build_test.go

```diff
@@ -477,5 +477,77 @@ var _ = Describe("Reconcile Build", func() {
 				Expect(reconcile.Result{}).To(Equal(result))
 			})
 		})
+
+		Context("when environment variables are specified", func() {
```

Comment:
Same as above, `valueFrom` is not considered here.

Reply:
updated in next push